fw-child/resources/scss/child.scss

```
@@ -652,13 +652,9 @@ $control-slider-handle-width: 2.75rem;
     font-size: 1rem;
   }
 
-  .fw-query-filter-reset {
-    color: $gray-600;
-    transition: 0.25s;
-
-    &:hover {
-      color: $secondary;
-    }
+  .fw-query-reset {
+    line-height: 0.75rem;
```

To be consistent with the "Clear" button used in the Maps and Download pages, we can add the following styles:
```suggestion
    line-height: 0.75rem;
    font-weight: 600;
    letter-spacing: 0.05em;
```

---

The clear button I see in Download data is font-weigth: 400.
Will be adressed in [#67062](https://3.basecamp.com/3451303/buckets/37454340/todos/7598541447)